code/modules/modular_computers/file_system/programs/nt_pay.dm

```diff
@@ -20,6 +20,9 @@
 /datum/computer_file/program/nt_pay/ui_act(action, list/params, datum/tgui/ui)
 	switch(action)
 		if("Transaction")
+			if(istype(current_user, /datum/bank_account/department))
+				return to_chat(usr, span_notice("The app is unable to withdraw from that card."))
```

Comment:
Currently we check the type for ``/obj/item/card/id/departmental_budget`` in these cases (like the roulette table).

We really should make a helper for this, maybe like ``#define IS_DEPARTMENTAL_ACCOUNT(card) (card in SSeconomy.dep_cards)``, and replace all instances with that (and preferably, this one too)

Reply:
Yeah thats a good idea wait a sec